pkg/api/meta/interfaces.go

```diff
@@ -107,7 +107,7 @@ const (
 )
 
 // RESTScope contains the information needed to deal with REST resources that are in a resource hierarchy
-// TODO After we deprecate v1beta1 and v1beta2, we can look a supporting removing the flexibility of supporting
+// TODO After we deprecate v1beta1 and v1beta2(deleted on June 1, 2015), we can look a supporting removing the flexibility of supporting
```

Comment:
Yes, this todo is done.  I deleted this comment.
I tested the following example, and the query param "?namespace=" doesn't work.
``` http://127.0.0.1:8080/api/v1/pods?namespace=aaa```
the result is:
~~~
{
  "kind": "PodList",
  "apiVersion": "v1",
  "metadata": {
    "selfLink": "/api/v1/pods",
    "resourceVersion": "17065"
  },
  "items": [
    {
      "metadata": {
        "name": "counter",
        "namespace": "default",
......
~~~